src/messaging.d.ts

```diff
@@ -673,13 +684,11 @@ export namespace admin.messaging {
     color?: string;
 
     /**
-     * Identifier used to replace existing notifications in the notification drawer.
-     *
-     * If not specified, each request creates a new notification.
-     *
-     * If specified and a notification with the same tag is already being shown,
-     * the new notification replaces the existing one in the notification drawer.
-     *
+     * The sound to be played when device receives a notification. Supports
```

Comment:
Since we are using articles everywhere else (instead of the extra-concise style that omits them), we should probably use them here too:

"The sound to be played when the device receives a notification. Supports
     *	     * "default" for the default notification sound of the device"